packages/desktop-gui/src/project/project.jsx

```diff
@@ -88,6 +89,15 @@ class Project extends Component {
   _reopenProject = () => {
     projectsApi.reopenProject(this.props.project)
   }
+
+  _pingBaseUrl = () => {
+    this.props.project.clearWarning(null, true)
```

Comment:
Passing `null` here clears all warnings, which I don't think this should do. It should just clear the `baseUrl` warning.